charts/osm/templates/osm-validator-webhook.yaml

```diff
@@ -39,5 +39,16 @@ webhooks:
       resources:
         - ingressbackends
         - egresses
+    {{- if .Values.smi.validateTrafficTarget }}
```

Comment:
This is going to conflict with my PR : https://github.com/openservicemesh/osm/pull/4160, we will have a the validator created in go if the reconciler is turned on. 

Could you rebase and make this change once I check in my PR ( I plan to in a few minutes)

Reply:
Based on my changes you will need to pass the validateTrafficTarget to the osm-controller and will have to add this additional logic to to the creation of the validating webhook go code

Reply:
updated. please take a look.

Reply:
thanks for this change, its good now. I had another comment